packages/bpk-component-accordion/src/BpkAccordionItem.js

```diff
@@ -144,7 +144,7 @@ BpkAccordionItem.defaultProps = {
   icon: null,
   onClick: () => null,
   tagName: 'span',
-  textStyle: TEXT_STYLES.base,
+  textStyle: TEXT_STYLES.bodyDefault,
```

Comment:
I'm thinking whether we should remove the weight from accordion completely as part of this ticket? Otherwise we are still using the text component with weight 🤔 what do you think?

Reply:
I think that might be something we should do separately, unless we know the users of the `weight` prop have already upgraded to remove that from their code?

Reply:
Looking at the code now, `weight` is still used in one project